source/mainnet/smart-contracts/guides/setup-contract.rst

```diff
@@ -15,79 +15,76 @@ The *from scratch* option guides you through the process when you want to start
 
    Concordium recommends that newcomers choose the *from a template* option.
 
-From a template
-===============
+.. dropdown:: From a template
 
-Concordium maintains several smart contract
-`templates <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/templates>`_ (currently a ``default`` template and a ``cis2-nft`` template).
-For generating the smart contracts from the above templates, the ``cargo-generate`` crate is required.
-``cargo-generate`` can be installed by running the following command:
+   Concordium maintains several smart contract
+   `templates <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/templates>`_ (currently a ``default`` template and a ``cis2-nft`` template).
+   For generating the smart contracts from the above templates, the ``cargo-generate`` crate is required.
+   ``cargo-generate`` can be installed by running the following command:
 
-.. code-block:: console
+   .. code-block:: console
 
-   $cargo install --locked cargo-generate
+      $cargo install --locked cargo-generate
 
-To start a new Concordium smart contract project from a template, run the command:
+   To start a new Concordium smart contract project from a template, run the command:
 
-.. code-block:: console
+   .. code-block:: console
 
-   $cargo concordium init
+      $cargo concordium init
 
-The path where the project should be created can be provided with the ``--path`` option.
+   The path where the project should be created can be provided with the ``--path`` option.
 
-You can find additional information on the available templates in the
-`README file <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/templates/README.md>`_.
+   You can find additional information on the available templates in the
+   `README file <https://github.com/Concordium/concordium-rust-smart-contracts/tree/main/templates/README.md>`_.
 
-From scratch
-============
+.. dropdown:: From scratch
```

Comment:
**Discussion:** I am not sure the main content should be in a dropdown like this. The initial experience on this page is some introduction paragraph and then a bunch of notes about wee-alloc.
Not sure what would be best, so this is to start the discussion on what could be done.
[Preview](https://developer.concordium.software/pr-preview/pr-963/en/mainnet/smart-contracts/guides/setup-contract.html)

Reply:
I agree, the dropdowns are not the best solution. I would suggest that we mention in the introduction that it can be done in two ways template/scratch which are links to further down on the page. And then there should be two sections, the from-template section, and the from-scratch section (and no dropdowns).